### DISCUSSION TITLE
Where can I find documentation about the different error codes for SSL_VERIFY_RESULT?

### DISCUSSION THREAD

**Discussion by GlassGruber** [curl/curl#14536] [Q&A] [answered]
Just as title, in the docs is stated that any non `0` returned code is an error but I was not able to find documentation about what are the possible codes.

**Comment by bagder** [comment, marked as answer]
That feature uses [CURLINFO_SSL_VERIFYRESULT](https://curl.se/libcurl/c/CURLINFO_SSL_VERIFYRESULT.html) under the hood. We do not document the different values. I don't think you should try to assign any particular meaning for them as curl makes no effort to - and it probably varies between TLS backends used.

**Reply by GlassGruber** [reply]
I see thank you! In any case I found that using `%{errormsg}` is kind of fine for the task.

PS. I really need to check better the libcurl api documentation, lots of good extra info!
